releasenotes/notes/55258.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+issue:
+- 55243
+releaseNotes:
+- |
+  **Added** support `--namespace` parameter for `istioctl experimental describe`.
```

Comment:
This is a regression rather than a new feature. Could you please revise this releasenote to avoid misleading users?

Reply:
Feel free to remove the hold once you've revised it.